samples/bootloader/pm.yml

```diff
@@ -1,12 +1,15 @@
 #include <autoconf.h>
 #include <generated_dts_board_unfixed.h>
 
-b0:
-  size: CONFIG_PM_PARTITION_SIZE_B0
+b0_image:
+  size: CONFIG_PM_PARTITION_SIZE_B0_IMAGE
   placement:
     after: start
     align: {end: 0x8000}
```

Comment:
While you are at it, you can remove this alignment. Since this is the first image in flash, we can not align it :) Also the end alignment does not makes ense when the provision is behind it. It should be the proviison partition that had the alignment requriment.

Reply:
Done